mtenn/conversion_utils/schnet.py

```diff
@@ -0,0 +1,111 @@
+"""
+Representation and strategy for SchNet model.
+"""
+from copy import deepcopy
+import torch
+from torch_geometric.nn.models import SchNet as PygSchNet
+
+from ..model import ConcatStrategy, DeltaStrategy, Model
+
+class SchNet(PygSchNet):
+    def __init__(self, model=None):
+        ## If no model is passed, construct default SchNet model, otherwise copy
+        ##  all parameters and weights over
+        if model is None:
+            super(SchNet, self).__init__()
+        else:
+            atomref = model.atomref.weight.detach().clone()
+            model_params = (model.hidden_channels, model.num_filters,
+                model.num_interactions, model.num_gaussians,
+                model.cutoff, model.max_num_neighbors,model.readout,
+                model.dipole, model.mean, model.std, atomref)
+            super(SchNet, self).__init__(*model_params)
+            self.load_state_dict(model.state_dict())
+
+    def forward(self, data):
+        return(super(SchNet, self).forward(data[0], data[1]))
+
+def get_representation(model):
```

Comment:
Would it be possible to use a class structure to localize all functions that operate on a model within a class scope, rather than file scope?

Reply:
I think that would work better!